packages/components/pie-switch/test/component/pie-switch.spec.ts

```diff
@@ -1,194 +1,196 @@
-import { test, expect } from '@sand4rt/experimental-ct-web';
-import { PieSwitch } from '../../src/index.ts';
-import {
-    type SwitchProps,
-    labelPlacements,
-    ON_SWITCH_CHANGED_EVENT,
-} from '../../src/defs.ts';
-
-const componentSelector = '[data-test-id="switch-component"]';
-const inputSelector = '[data-test-id="switch-input"]';
-const switchLabelSelector = (placement: SwitchProps['labelPlacement'] = 'leading') => `[data-test-id="switch-label-${placement}"]`;
+import { test, expect } from '@playwright/test';
+import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+import { labelPlacements, type SwitchProps } from '../../src/defs';
+import { pieSwitch } from '../helpers/page-objects/selectors';
 
 test.describe('Component: `Pie switch`', () => {
-    test.beforeEach(async ({ mount }) => {
-        const component = await mount(PieSwitch);
-        await component.unmount();
-    });
-
-    test('should be visible', async ({ mount, page }) => {
+    test('should be visible', async ({ page }) => {
         // Arrange
-        await mount(PieSwitch);
+        const switchPage = new BasePage(page, 'switch');
+        await switchPage.load();
 
         // Act
-        const pieSwitch = page.locator(componentSelector);
+        const switchComponent = page.getByTestId(pieSwitch.selectors.container.dataTestId);
 
         // Assert
-        await expect(pieSwitch).toBeVisible();
+        await expect(switchComponent).toBeVisible();
     });
 
-    test('should have a "visible" input to help with accessibility', async ({ mount, page }) => {
+    test('should have a "visible" input to help with accessibility', async ({ page }) => {
         // Arrange
-        await mount(PieSwitch);
+        const switchPage = new BasePage(page, 'switch');
+        await switchPage.load();
 
         // Act
-        const input = page.locator(inputSelector);
+        const input = page.getByTestId(pieSwitch.selectors.input.dataTestId);
 
         // Assert
         await expect(input).toBeVisible();
     });
```

Comment:
I know you didn't write this but we need to assess these tests as we go through them, otherwise they'll never improve. I am not sure what "visible" in quotes here means and "help with accessibility" is quite vague. Can we please either remove this test or rename? I'm not sure what value this assertion is providing